File: OpenAlprWebhookProcessor/angularapp/src/app/settings/alerts/pushover/pushover.ts
export class Pushover {
    apiToken: string;
    userKey: string;
    isEnabled: boolean;
    sendPlatePreviewEnabled: boolean;
    sendEveryPlateEnabled: boolean;
    
    constructor(init?:Partial<Pushover>) {
        Object.assign(this, init);
    }
}

File: OpenAlprWebhookProcessor/Data/Pushover.cs
using System;

namespace OpenAlprWebhookProcessor.Data
{
    public class Pushover
    {
        public Guid Id { get; set; }

        public bool IsEnabled { get; set; }

        public string UserKey { get; set; }

        public string ApiToken { get; set; }

        public bool SendPlatePreview { get; set; }

        public bool SendEveryPlateEnabled { get; set; }
    }
}
